.github/workflows/stale.yml

```diff
@@ -18,6 +18,8 @@ jobs:
           days-before-pr-stale: 30
           days-before-pr-close: 10
           stale-pr-label: "stale"
+          exempt-issue-labels: "do-not-stale"
+          exempt-pr-labels: "do-not-stale"
```

Comment:
Based on line 15, the stale bot currently does not close issues.

The source of the problem is Dosubot, which is currently closing issues that we don't want to be closed.  The issue is we're not configuring Dosubot here.

I contacted the Dosubot team to see if we could disable this feature, and they will make this happen:
* Mark as stale, but do not close for all issues